OPHD/ophd.vcxproj

```diff
@@ -105,6 +105,16 @@
       <TargetMachine>MachineX86</TargetMachine>
       <AdditionalLibraryDirectories>$(SolutionDir)$(Configuration)\;%(AdditionalLibraryDirectories)</AdditionalLibraryDirectories>
     </Link>
+    <PostBuildEvent>
+      <Command>SET sourceDir="$(VcpkgCurrentInstalledDir)bin\"
+if $(ConfigurationName) == Debug (
+  SET sourceDir="$(VcpkgCurrentInstalledDir)debug\bin\"
+)

```

Comment:
I suspect this can be simplified by using `VcpkgConfigSubdir`, which is conditionally either empty `''`, or `'debug\'`. That would allow us to remove the conditional block of our own.

https://github.com/microsoft/vcpkg/blob/544f8e4593764f78faa94bac2adb81cca5232943/scripts/buildsystems/msbuild/vcpkg.targets#L65

Effectively we could simplify this to:
```
SET sourceDir="$(VcpkgCurrentInstalledDir)$(VcpkgConfigSubdir)bin\"
```

Reply:
I'm not sure that VcpkgConfigSubdir is working as advertised? You should be able to see the expansion below. Maybe the spelling is wrong or something else?

<img width="571" alt="VcpkgConfigSubdir Not Working" src="https://user-images.githubusercontent.com/15183337/87214907-e2286000-c2fe-11ea-8047-4fb66eb1ca28.png">

Reply:
Huh, fascinating. I wonder if it might be a recently added variable that perhaps doesn't exist in the copy you're using.

Probably best not to rely on it if it's not likely to work for other people.